### ISSUE TITLE
Frontend: initial screen reader integration

### ISSUE THREAD

**Issue by GeorgeZudikhin** [GeorgeZudikhin/autogenerated-directions-FHTW#33]
As a visually impaired or blind user,
I want the website to have an implementation of a screen reader,
So that the I can rely on the screen reader to inform me of the contents of the website.

### Acceptance Criteria:

- [x] Screen Reader technology researched and planned.
- [x] Proof of concept: the screen reader should be integrated into the website, so that its functionality can be at least preliminarily tested.
